test/storage/conftest.py

```diff
@@ -17,10 +17,10 @@
 from test.cluster.conftest import *
 
 
-# Mounting volumes outside of the toolchain environment, requires root privileges. To overcome it,
-# `unshare` is used to launch tests in a separate namespace. This is a temporary solution as it
-# works with test.py but breaks the execution of the tests with pytest without test.py.
-# FIXME: Find a more robust solution to ditch unshare.
+# Mounting volumes outside of the toolchain environment, requires root privileges.
+# FIXME: Cannot use unshare as the test launcher because the cluster is created before
+# the tests are executed. That means, that adding servers happens also outside of the
```

Comment:
Can't we pass the unshare requirement to the fixture?

Reply:
The cluster is created with a call to the cluster manager API. Add a namespace parameters to the fixture, which would be passed to the cluster manager, which would then create the namespace and the cluster.

Or even: unconditionally create clusters in namespaces and cgroups, we need that to collect runtime data.

Reply:
The current path is to remove this http server that manages the cluster creation, and let the fixtures create the cluster. That's why I would rather not extend the solution that we are trying to eliminate. Moreover, this is a workaround while the solution without `sudo` is under development.